### PR TITLE
Update local configuration info

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Review by shawnbmccarthy — APPROVED** [review]
LGTM

File: docs/manage/configuration.md [hunk 1]
```diff
@@ -317,14 +340,61 @@ If you run into issues, here are some things to try:
...
+```
 
-You can write JSON manually, or you can use the Viam app's config builder (without connecting your robot to it) to generate JSON.
-Once you finish configuring in **Builder** mode, switch to **Raw JSON** and copy the entire raw JSON into your <file>/etc/viam.json</file>.
+{{% /tab %}}
+{{< /tabs >}}
 
-Technically, you can put your config file anywhere you want as long as when starting `viam-server`, you pass in the path to the config file.
-<file>/etc/viam.json</file> is just the default location.
+To run `viam-server` as a system service, see [Installing as a System Service](/installation/install/linux-install/#installing-as-a-system-service).
```
Comment (npentrel) [inline docs/manage/configuration.md:397]:
```suggestion
To run `viam-server` as a system service, see [Start automatically on boot](/installation/#start-automatically-on-boot).
```

File: docs/manage/configuration.md [hunk 2]
```diff
@@ -21,11 +21,34 @@ The credentials look like this:
...
 
+{{< alert title="Tip" color="tip" >}}
+You can also store your config file in another location.
+To use such a config file, pass in the path to the config when you start `viam-server`:
+
+{{< tabs >}}
+{{% tab name="Linux" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo viam-server -config <path-to-config>.json
```
Comment (JessamyT) [inline docs/manage/configuration.md:37]: Does this actually start the server? Or does this change where it'll look for its config for ever more in the future?

File: docs/manage/configuration.md [hunk 3]
```diff
@@ -21,11 +21,34 @@ The credentials look like this:
...
 
-When you turn on your robot, `viam-server` starts up and uses the credentials in its local <file>/etc/viam.json</file> to fetch its full config from the [Viam app](https://app.viam.com).
+When you turn on your robot, `viam-server` starts up and, by default, it uses the credentials in its local <file>/etc/viam.json</file> to fetch its full config from the [Viam app](https://app.viam.com).
 Once the robot has a configuration, it caches it locally and can use the configuration for up to 60 days.
 The robot checks for new configurations every 15 seconds and changes its configuration automatically when a new configuration is available.
 All communication happens securely over HTTPS using secret tokens that are in a robot's configuration.
 
+{{< alert title="Tip" color="tip" >}}
+You can also store your config file in another location.
+To use such a config file, pass in the path to the config when you start `viam-server`:
```
Comment (JessamyT) [inline docs/manage/configuration.md:31]:
```suggestion
To change where `viam-server` looks for its config file, run the following command:
```
Maybe?

File: docs/manage/configuration.md (hunk 2 above)
Comment (npentrel) [reply docs/manage/configuration.md:37]: it just starts the server once with that config

**Pull request merged**